lighthouse-core/lib/url-shim.js

```diff
@@ -59,9 +59,16 @@ URL.hostsMatch = function hostsMatch(urlA, urlB) {
 
 /**
  * @param {string} url
+ * @param {{pathParts: number, removeQuery: boolean, removeHost: boolean}=} options
  * @return {string}
  */
-URL.getDisplayName = function getDisplayName(url) {
+URL.getDisplayName = function getDisplayName(url, options) {
+  options = Object.assign({
+    pathParts: 2,
+    removeQuery: true,
+    removeHost: true,
```

Comment:
I've been slowly sneaking them in over time as I touch files :) I'm in favor of always enforcing for cleaner diffs